src/main/app/claims/ccjClient.ts

```diff
@@ -0,0 +1,17 @@
+import User from 'app/idam/user'
+import request from 'client/request'
+import { CCJModelConverter } from 'claims/ccjModelConverter'
+import { claimStoreApiUrl } from 'claims/claimStoreClient'
+
+export class CCJClient {
+  static save (user: User): Promise<object> {
```

Comment:
Can we set a more precise return type on this?

Reply:
yes